wire-schema-tests/build.gradle.kts

```
@@ -15,20 +13,112 @@ if (project.rootProject.name == "wire") {
   apply(plugin = "binary-compatibility-validator")
 }
 
-dependencies {
-  api(libs.junit)
-  api(projects.wireCompiler)
-  api(projects.wireSchema)
-  implementation(projects.wireKotlinGenerator)
-  implementation(projects.wireJavaGenerator)
-  implementation(projects.wireSwiftGenerator)
-  implementation(libs.okio.core)
-  implementation(libs.okio.fakefilesystem)
-  testImplementation(libs.assertj)
-  testImplementation(libs.kotlin.test.junit)
-  testImplementation(projects.wireTestUtils)
-}
+kotlin {
+  jvm {
+    withJava()
+  }
```

```suggestion
  jvm()
```
I don't see any Java sources in here, right?

---

Looks like `MavenPublishBaseExtension` fails without it with `SoftwareComponentInternal with name 'java' not found`